pkg/resources/validation/cluster/provisioningcluster.go

```diff
@@ -22,63 +22,64 @@ type provisioningClusterValidator struct{}
 func (p *provisioningClusterValidator) Admit(response *webhook.Response, request *webhook.Request) error {
 	listTrace := trace.New("provisioningClusterValidator Admit", trace.Field{Key: "user", Value: request.UserInfo.Username})
 	defer listTrace.LogIfLong(2 * time.Second)
-	cluster, err := clusterObject(request)
+	oldCluster, cluster, err := objectsv1.ClusterOldAndNewFromRequest(request)
 	if err != nil {
 		return err
 	}
 
+	if response.Result, err = validateCreatorID(request, oldCluster, cluster); err != nil || response.Result != nil {
+		return err
+	}
+
+	if response.Result, err = validateACEConfig(cluster); err != nil || response.Result != nil {
+		return err
+	}
+
+	response.Allowed = true
+	return nil
+}
+
+func validateCreatorID(request *webhook.Request, oldCluster, cluster *v1.Cluster) (*metav1.Status, error) {
```

Comment:
Neither function returns or has the ability to return an error so it doesn't need to be here

Reply:
Good catch!